src/logging.c

```diff
@@ -803,23 +883,18 @@ void rcutils_logging_console_output_handler(
       "call to rcutils_logging_console_output_handler failed.\n");
     return;
   }
-  FILE * stream = NULL;
   switch (severity) {
```

Comment:
@clalancette what's the point of the `switch` statement now? to ensure the given severity level is valid?

Reply:
Yeah, exactly.  Once I restore the `return` in the default case, it is just error checking.